storage/mysql/map_storage_test.go

```diff
@@ -99,20 +99,23 @@ func TestMapSnapshot(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.desc, func(t *testing.T) {
 			tx, err := s.SnapshotForTree(ctx, test.tree)
-
-			if hasErr := err != nil; hasErr != test.wantErr {
-				t.Fatalf("%v: err = %q, wantErr = %v", test.desc, err, test.wantErr)
-			} else if hasErr {
-				return
+			if err != nil {
+				t.Fatalf("SnapshotForTree()=_,%v; want _, nil", err)
 			}
 			defer tx.Close()
 
 			_, err = tx.LatestSignedMapRoot(ctx)
 			if err != nil {
```

Comment:
This logic would be a bit simpler if we used `got` / `want`

Reply:
Fixed.